src/System.Net.HttpListener/src/System/Net/HttpListenerTimeoutManager.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net
+{
+    public partial class HttpListenerTimeoutManager
+    {
+    }
```

Comment:
Is the idea here that common stuff from the Windows and eventual Unix implementations will be factored out to here?  Otherwise this isn't needed and the type doesn't need to be partial.